FreddysBBQ/src/OrderService/run_tests.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
This will run contract tests against a running app. I didn't know how to make it part of the build